cmd/lock-rpc-server.go

```diff
@@ -20,7 +20,7 @@ import (
 	"fmt"
 	"math/rand"
 	"net/rpc"
-	"path"
+	pathutil "path"
```

Comment:
a struct having `path` field is not a problem but a variable naming `path` will confuse in general.  That should be avoided.